src/index.html

```diff
@@ -4,9 +4,329 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#home" class="logo-link">
```

Comment:
Нащо робити окремий блок для ссилки?
```suggestion
          <a href="#home" class="header__logo-link">
```

Reply:
+